scripts/migrate-to-0.7

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import argparse
+import json
+import sys
+
+
+class MigrationFailed(Exception):
+    def __init__(self, msg):
+        super(MigrationFailed, self).__init__(msg)
+
+
+def migrate_node(node):
+    """
+    Migrate a JSON node to from our old format to the new format. This will
+    only migrate the node to the new format if it is naively compatible - that
+    is the old version was just encoding a single feature, single predicate
+    binary split.
+    """
+    if type(node) == list:
+        if len(node) != 2:
+            raise MigrationFailed('split node is not binary')
+        left, right = node
+        key = left['feature']
+        if right['feature'] != key:
+            raise MigrationFailed('predicates use different feature')
+        predicate = left['predicate']
+        if right['predicate'].get('not') != predicate:
```

Comment:
Yeah - I was thinking about this. None of the default splitters would do anything smart here, but it's possible someone may have a fancier splitter that does. It certainly doesn't hurt to handle the symmetric case anyways!